core/object/object.cpp

```diff
@@ -863,28 +863,37 @@ Variant Object::call_const(const StringName &p_method, const Variant **p_args, i
 
 void Object::notification(int p_notification, bool p_reversed) {
 	if (p_reversed) {
+		_notification_fast<true>(p_notification);
+	} else {
+		_notification_fast<false>(p_notification);
+	}
+}
+
+template <bool REVERSED>
```

Comment:
```suggestion
template <bool Reversed>
```
Or
```suggestion
template <bool reversed>
```
We don't use all upper case for template arguments

Reply:
We're actually not consistent on this. Across the codebase we're using many different styles. My suggestion of using all-upper is as valid as any other that makes the template argument stand out. I've seen `t_`-prefixed template arguments in some files. I'd advocate for that: `t_prefixed` and we can do a later codebase-wide harmonization. That said, I don't mind merging this with `reversed`, as long as the consistency pass happens at some point.